tools/tertiary-analysis/atlas-data-import/atlas_import_experiment_data.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="atlas_import_experiment_data" name="Atlas import: get experiment data" version="@TOOL_VERSION@+galaxy0"  profile="@PROFILE@">
```

Comment:
This should be all options of the main downloader I would say. Otherwise we clutter with an unnecessarily high number of tools.